molgenis-data-rest/src/main/java/org/molgenis/data/rest/v2/RestControllerV2.java

```diff
@@ -465,6 +534,32 @@ public ErrorMessageResponse handleValidationException(MolgenisValidationExceptio
 		return new ErrorMessageResponse(new ErrorMessage(e.getMessage()));
 	}
 
+	@ExceptionHandler(MolgenisRepositoryCapabilitiesException.class)
+	@ResponseStatus(BAD_REQUEST)
+	@ResponseBody
+	public ErrorMessageResponse handleMolgenisRepositoryCapabilitiesException(MolgenisRepositoryCapabilitiesException e)
+	{
+		LOG.info("Molgenis repository capabilities exception occurred.", e);
+		return new ErrorMessageResponse(new ErrorMessage(e.getMessage()));
+	}
+
+	@ExceptionHandler(DuplicateEntityException.class)
+	@ResponseStatus(BAD_REQUEST)
+	@ResponseBody
+	public ErrorMessageResponse handleDuplicateEntityException(DuplicateEntityException e)
+	{
+		LOG.info("Duplicate entity exception occurred.", e);
+		return new ErrorMessageResponse(new ErrorMessage(e.getMessage()));
+	}
+
+	@ResponseStatus(BAD_REQUEST)
+	@ResponseBody
+	public ErrorMessageResponse handleUnknownEntityException(UnknownEntityException e)
```

Comment:
I have thought the same, but it will break other functionality.
Sometimes the application throws other MolgenisDataException exceptions. I don't want to manage all of them